helm/v1beta1/0000_10-project-helm-chart-repository.crd.yaml

```diff
@@ -53,6 +53,9 @@ spec:
                         name:
                           description: name is the metadata.name of the referenced config map
                           type: string
+                        namespace:
+                          description: namespace is the metadata.namespace of the referenced config map
+                          type: string
                     tlsClientConfig:
                       description: tlsClientConfig is an optional reference to a secret by name that contains the PEM-encoded TLS client certificate and private key to present when connecting to the server. The key "tls.crt" is used to locate the client certificate. The key "tls.key" is used to locate the private key. The namespace for this secret is openshift-config.
```

Comment:
change this description at the end saying the namespace will be openshift-config. The namespace in this case is what the use enters otherwise defaulted to the namespace where the namespace scoped repo is instantiated.

Reply:
Done